engine/uState.ml

```diff
@@ -953,6 +936,11 @@ let check_constraint uctx (c:UnivProblem.t) =
         | QConstant QProp, QVar q -> QState.is_above_prop uctx.sort_variables q
         | (QConstant _ | QVar _), _ -> false
       end
+  | QElimTo (a,b) ->
```

Comment:
Spaces are cheap nowadays. It's not because the surrounding code has bad taste that you have to stick to it...